### PR TITLE
Deliver changes from Contribution Day

### PR DESCRIPTION
**Pull request by okolesnyk**
### Description
To verify basic Framework functionality I want to be able to create and run MFTF tests without any other projects required.
Included Pull Requests from Contribution Day. Please refer to issues to get more info about.

### Fixed Issues (if relevant)
1. Resolve magento/magento2-functional-testing-framework#56: Add "block" type to list of existing section element types
2. Resolve magento/magento2-functional-testing-framework#57: Debug flag exists in robo generate:tests to print action sequence of test steps
3. Resolve magento/magento2-functional-testing-framework#58: Allow browser type to be configured in env file
4. Resolve magento/magento2-functional-testing-framework#60: Build interactive configuration tool for populating .env properties
5. Resolve magento/magento2-functional-testing-framework#61: Validate build:project was run before attempting to generate tests

### Contribution checklist
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/verification tests (if applicable)
 - [x] All automated tests passed successfully (all builds on Travis CI are green)
 - [x] Changes to Framework doesn't have backward incompatible changes for tests or have related Pull Request with fixes to tests

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2-functional-testing-framework?pullRequest=63) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/magento2-functional-testing-framework?pullRequest=63) before we can accept your contribution.<br/>

File: RoboFile.php [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+            $sg->generateSuite($arg);
+        }
+    }
+
+    /**
+     * Run all Functional tests.
+     *
+     * @return void
+     */
+    function functional()
```
Comment (tomreece) [inline RoboFile.php]: This robo command did not work as expected.

File: src/Magento/FunctionalTestingFramework/Util/Env/EnvProcessor.php [hunk 2]
```diff
@@ -0,0 +1,99 @@
+<?php
```
Comment (imeron2433) [inline src/Magento/FunctionalTestingFramework/Util/Env/EnvProcessor.php]: We have a similar type of class as a 3rd party library, why did we write a new one?

Comment (imeron2433) [reply src/Magento/FunctionalTestingFramework/Util/Env/EnvProcessor.php]: Discussed, future implementaion will get rid of this dependency.

**Pull request merged**